process/resultsParser/returnCode.go

```diff
@@ -0,0 +1,27 @@
+package resultsParser
+
+import (
+	"bytes"
+	"encoding/hex"
+)
+
+// ReturnCode is an enum that describes some common smart contract results.
+type ReturnCode string
+
+const (
```

Comment:
please remove these consts already defined in `mx-chain-vm-common-go`

Reply:
changed.